python/azureml/containerInstance/ArmTemplates/template.json

```diff
@@ -34,6 +34,13 @@
         },
         "appInsightsLocation": {
             "type": "string"
+        },
+        "newOrExistingKeyVault": {
+            "type": "string", 
+            "allowedValues": [
```

Comment:
how will this value change, now?

Reply:
@sachinma 
Missed to mention that this PR is meant for suggesting the way if we want to use Condition. Agreed that this will not gonna pull up in our scenario.
Provide your feedback if you can think of any possible solution via Condition!